pom.xml

```diff
@@ -250,6 +262,13 @@ THE SOFTWARE.
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>org.jenkins-ci.main</groupId>
+                <artifactId>jenkins-bom</artifactId>
```

Comment:
wasn't the whole point of bom to stop having to specify specific plugin versions?

Reply:
Yes; the version overrides specified in pom.xml can be removed with this bom applied.

Reply:
Removing the versions from all plugin deps does not work for me. I also couldn't find a reference that this should work in neither bom code ( core-bom and plugin-bom )

Reply:
@res0nance is this topic holding back further reviews and approvals?